src/debugging-support-in-rustc.md

```diff
@@ -104,6 +104,27 @@ LLDB has Rust-like value and type output.
 * LLDB has a plugin architecture but that does not work for language support.
 * GDB generally works better on Linux.
 
+### WinDbg/CDB
+
+Microsoft provides [Windows Debugging Tools] such as the Windows Debugger (WinDbg) and
+the Console Debgugger (CDB) which both support debugging programs written in Rust. These
+debuggers parse the debug info for a binary from the `PDB`, if available, to construct a
+visualization to serve up in the debugger.
+
+#### Natvis
+
+Both WinDbg and CDB support defining and viewing custom visualizations for any given type
+within the debugger using the Natvis framework. The Rust compiler defines a set of Natvis
+files that define custom visualizations for a subset of types in the standard libraries such
+as, `std, core and alloc`. These Natvis files are embedded into `PDBs` generated by the
+`*-pc-windows-msvc` target triples to automatically enable these custom visualizations when
+debugging. This default can be overridden by setting the `strip` rustc flag to either `debuginfo`
+or `symbols`.
```

Comment:
Oh, interesting! I wasn't aware of that possibility. We may want to document this in the rustc book if this is generally useful (given that we separate debug info from code on Windows, it doesn't seem super important in common scenarios, but I could certainly be missing something).

Reply:
Yes, Natvis is only embedded if the `strip` rustc flag is omitted or set to `none`, which is the default.

> We may want to document this in the rustc book

Where would this best be documented? I don't really know where that might fit in the stable Rust book.

Reply:
This might be a good spot https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/codegen-options/index.md#strip

Reply:
> This might be a good spot https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/codegen-options/index.md#strip

Sounds good, since this will be going in a different repo, I'll resolve this comment here. Thanks!